src/servers/ZoneServer2016/managers/constructionmanager.ts

```diff
@@ -352,13 +352,16 @@ export class ConstructionManager {
     return false;
   }
   detectPOIPlacement(
+    server: ZoneServer2016,
```

Comment:
idk why you keep adding the whole server as an argument when you can just ignore the function call based on your boolean

Reply:
It was to provide clarity for people learning. You look for the isNoBuildInPoi variable and see it's not even in the detectPOIPlacement when you would expect it to be. Is it gonna completely break the code for using the server instance or something 😂😂?